src/Symfony/Component/Serializer/Normalizer/FlattenExceptionNormalizer.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\ErrorHandler\Exception\FlattenException;
+use Symfony\Component\Serializer\Exception\InvalidArgumentException;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+
+/**
+ * Normalizes FlattenException instances.
```

Comment:
I would add a warning about the danger of using it outside of a debugging/development context.

Reply:
This is currently used by Messenger to serialize exceptions into the Transport Headers (e.g. The error happens when you serialize to json the exception). Should I add that warning still?